code/__HELPERS/mobs.dm

```diff
@@ -128,7 +130,8 @@
 		"apid_stripes" = pick(GLOB.apid_stripes_list),
 		"apid_headstripes" = pick(GLOB.apid_headstripes_list),
 		"body_model" = gender == MALE ? MALE : gender == FEMALE ? FEMALE : pick(MALE, FEMALE),
-		"psyphoza_cap" = pick(GLOB.psyphoza_cap_list)
+		"psyphoza_cap" = pick(GLOB.psyphoza_cap_list),
+		"tail_monkey" = "None",
```

Comment:
Monkeys aren't going to be a roundstart species, so we don't need to hook into players preferences. So no SQL, I think.

I will look into the feature, since now that I think about it, this should just be a regular tail on default_features instead of a snowflake feature

Reply:
Make sure to setup a local database and test that saving characters actually works. I think character saves will fail if something is present in this list but isn't in the database as the insert query will be malformed.

Reply:
c29a190 Removed the new entry, shouldnt need to update schema anymore